dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/JdbcEventStore.java

```diff
@@ -147,6 +145,27 @@
 @Repository( "org.hisp.dhis.dxf2.events.event.EventStore" )
 public class JdbcEventStore implements EventStore
 {
+
+    private static final String COMMENT_QUERY =
```

Comment:
minor: since this query looks like is only for events, I would rename to PSI_EVENT_QUERY or something like this

Reply:
this one is the sub-query for comments, so I'll rename to PSI_EVENT_COMMENT_QUERY